h/assets.py

```diff
@@ -85,12 +85,16 @@ def urls(self, bundle):
         Returns the URLs at which all files in a bundle are served,
         read from the asset manifest.
         """
-        manifest = self.manifest.load()
         bundles = self.bundles.load()
 
-        def asset_url(path):
-            return '{}/{}'.format(self.assets_base_url, manifest[path])
-        return [asset_url(path) for path in bundles[bundle]]
+        return [self.asset_url(path) for path in bundles[bundle]]
+
+    def asset_url(self, path):
+        """
+        Return the cache-busted URL for an asset with a given path.
+        """
+        manifest = self.manifest.load()
```

Comment:
I think we want to be a little careful here, as previously we were doing one call to `load()` per bundle, and now we're doing one call to load per file (including when those files are referenced from a bundle).

The manifest is cached, sure, but this is still going to do at least one `stat()` call, which isn't ideal if we're doing it dozens of times during a single request. Especially in production, where we know the manifest isn't going to change, it might be nice to load it just once.